### PR TITLE
APP-583

### PR DESCRIPTION
**Pull request by robson-sanchez**
- Password passing in cleartext on command line
- Remove hardcoded default password / plaintext password in source

### CODE REVIEW

**Comment by rsanchez-daitan** [comment]
ok to test

**Review by miltongq — APPROVED** [review]
LGTM

**Comment by robson-sanchez** [comment]
@dnsymphony please check

File: integration-commons/src/main/java/org/symphonyoss/integration/BaseIntegration.java [hunk 1]
```diff
@@ -126,15 +126,15 @@ public void registerUser(String integrationUser) {
...
     Application application = properties.getApplication(integrationUser);
 
-    Keystore keystoreConfig;
-    if (application == null) {
-      keystoreConfig = new DefaultAppKeystore(integrationUser);
-    } else if (application.getKeystore() == null) {
-      keystoreConfig = new DefaultAppKeystore(application.getId());
-    } else {
-      keystoreConfig = application.getKeystore();
+    if ((application == null) || (application.getKeystore() == null) || (StringUtils.isEmpty(
```
Comment (dnsymphony) [inline integration-commons/src/main/java/org/symphonyoss/integration/BaseIntegration.java]: Line break on ||

**Review by dnsymphony — APPROVED** [review]

**Pull request merged**